typescript/examples/langchain-cdp-chatbot/.prettierignore

```diff
@@ -0,0 +1,7 @@
+docs/
```

Comment:
These files don't need to be copy-pasted into each package, but I'm holding off on creating a shareable config until we lock down the monorepo management tool we end up going with